ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -10,11 +10,54 @@ import LlamaKit
 import Nimble
 import Quick
 import ReactiveCocoa
+import Foundation
+
+private func startSignalProducer<T: Equatable, E: Equatable>(signalProducer: SignalProducer<T, E>, #expectSentValue: T?, sentError expectSentError: E?, #complete: Bool) {
+	startSignalProducer(signalProducer, expectSentValues: expectSentValue.map { [$0] } ?? [], sentError: expectSentError, complete: complete)
+}
+
+private func startSignalProducer<T: Equatable, E: Equatable>(signalProducer: SignalProducer<T, E>, expectSentValues: [T] = [], sentError expectSentError: E?, #complete: Bool) {
```

Comment:
I initially implemented all of this in each method, and after a while I realized it was pretty WET. This method made the tests very concise and easy to read: they just describe what the expected output from the producer is.

That been said, I'm gonna look into implementing it as a matcher and see where that takes me :+1: